server/sonar-server/src/main/java/org/sonar/server/ws/WsParameterBuilder.java

```diff
@@ -85,7 +89,12 @@ private static String buildQualifiersDescription(QualifierParameterContext conte
 
   private static String qualifierLabel(QualifierParameterContext context, String qualifier) {
     String qualifiersPropertyPrefix = "qualifiers.";
-    return context.getI18n().message(context.getUserSession().locale(), qualifiersPropertyPrefix + qualifier, "");
+    String qualifierDescription = context.getI18n().message(context.getUserSession().locale(), qualifiersPropertyPrefix + qualifier, "no description available");
+    if (DEPRECATED_QUALIFIERS.contains(qualifier)) {
+      // even if the qualifier in the API was deprecated before that, the WS documentation has been updated in 5.6
+      qualifierDescription += " - deprecated since 5.6";
```

Comment:
- api/components/tree
- api/components/search (I've just realized that its documentation may not have been updated because it doesn't call WsParametersBuilder)